app/controllers/application_controller/ci_processing.rb

```diff
@@ -1852,8 +1852,15 @@ def process_cloud_object_storage_buttons(pressed)
   end
 
   def cloud_object_store_button_operation(klass, task)
-    method = "#{task}_#{klass.name.underscore.to_sym}"
-    display_name = _(task.capitalize)
+    # Map to instance method name
+    case task
+    when "delete"
+      method = "#{task}_#{klass.name.underscore.to_sym}"
+      display_name = _(task.capitalize)
+    else
+      display_name = _(task.capitalize)
+      method = task = "#{klass.name.underscore.to_sym}_#{task}"
+    end
 
```

Comment:
A change in this file is needed due to naming inconsistency in the backend:
* for operation `params[:pressed] == "cloud_object_store_container_delete"` the supports? task name is ":delete" and method to be executed is "delete_cloud_object_store_container"
* for operation `params[:pressed] == "cloud_object_store_container_clear"` the supports? task name is ":cloud_object_store_container_clear" and method to be executed is "cloud_object_store_container_clear".

I expect other operations to follow second convention therefore I made it default.